lib/open_project/translations/models/locales_updater_configuration.rb

```diff
@@ -0,0 +1,14 @@
+require 'yaml'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.